common/app/views/fragments/standfirst.scala.html

```diff
@@ -16,7 +16,7 @@
     @content.standfirst.map { s =>
         @defining(Edition(request)) { edition =>
             @withJsoup(BulletCleaner(
-                if(content.isImmersive && content.isArticle) {
```

Comment:
assuming byline is an `Option`, it would possibly be clearer to 
```
content.byline.map { byline => 
 if (s.contains(byline)) {
   [..]
 } else {
   s + "<p>Words by" + ContributorLinks(byline, content.contributors) + "</p>"
 }
```
I'd also suggest to rename s to standfirst while if you're in there